nixos/tests/kanidm.nix

```diff
@@ -42,17 +55,17 @@ import ./make-test-python.nix ({ pkgs, ... }:
 
       environment.etc."kanidm/config".text = ''
         uri = "https://${serverDomain}"
-        verify_ca = true
-        verify_hostnames = true
       '';
```

Comment:
What does this configure? Shouldn't this part be configured through the kanidm client module bits?

Reply:
There’s not really a client module at this moment, so the config file only gets created if a server or pam support is enabled :/

Oh, and these verify settings are `true` by default, so I removed them.

Reply:
I split the options into `enableClient` and `enableServer`, so the test can use `clientSettings` now.